app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -526,6 +322,52 @@ class TopicLessonsFragmentTest {
     }
   }
 
+  private fun createTopicActivityIntent(internalProfileId: Int, topicId: String): Intent {
+    return TopicActivity.createTopicActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      internalProfileId,
+      topicId
+    )
+  }
+
+  private fun clickLessonTab() {
+    testCoroutineDispatchers.runCurrent()
+    onView(
+      allOf(
+        withText(TopicTab.getTabForPosition(1).name),
+        isDescendantOfA(withId(R.id.topic_tabs_container))
+      )
+    ).perform(click())
+    testCoroutineDispatchers.runCurrent()
+  }
+
+  private fun clickStoryItem(position: Int, targetViewId: Int) {
+    onView(
+      atPositionOnView(
+        R.id.story_summary_recycler_view,
+        position,
+        targetViewId
+      )
+    ).perform(click())
```

Comment:
Suggest adding another runCurrent() after this to help guard against stabilization issues (you may not notice them, but they can crop up for other people).

Reply:
done